Tasks/Common/ios-signing-common/ios-signing-common.ts

```diff
@@ -132,6 +132,52 @@ export async function findSigningIdentity(keychainPath: string) {
     }
 }
 
+/**
+ * Check if provisioning profile contains Cloud entitlement 
+ * @param provProfilePath
+ * @returns {boolean} 
+ */
+export async function cloudEntitlement(provProfilePath: string) {
+    //find the provisioning profile details
+    let provProfileDetails: string;
+    let getProvProfileDetailsCmd: ToolRunner = tl.tool(tl.which('security', true));
+    getProvProfileDetailsCmd.arg(['cms', '-D', '-i', provProfilePath]);
+    getProvProfileDetailsCmd.on('stdout', function (data) {
+        if (data) {
+            if (provProfileDetails) {
+                provProfileDetails = provProfileDetails.concat(data.toString().trim().replace(/[,\n\r\f\v]/gm, ''));
+            } else {
+                provProfileDetails = data.toString().trim().replace(/[,\n\r\f\v]/gm, '');
+            }
+        }
+    })
+    
+    await getProvProfileDetailsCmd.exec();
+
+    let tmpPlist: string;
+    if (provProfileDetails) {
+        //write the provisioning profile to a plist
+        tmpPlist = '_xcodetasktmp.plist';
```

Comment:
Where does this file get written? Should it not use a temporary or working directory? What happens if it already exists? Should it use some kind of random suffix?

Reply:
If it exists it will just overwrite it. 
The file is written in the local directory where the service is running.